src/main/java/core/basesyntax/controller/ControllerHandler.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.models.BetModel;
+import java.util.Scanner;
+
+public class ControllerHandler {
+    private BetDao betDao = new BetDaoImpl();
+
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase("q")) {
+                return;
+            }
+            BetModel bet = null;
+            try {
+                String[] betData = command.split(" ");
+                int betValue = Integer.parseInt(betData[0]);
+                double betRisk = Double.parseDouble(betData[1]);
+                bet = new BetModel(betValue, betRisk);
+
+            } catch (NumberFormatException e) {
```

Comment:
add ArrayIndexOutOfBoundsException to this catch block as well

Reply:
added